addons/l10n_it_edi_sdicoop/tests/test_edi_reverse_charge_xml.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import datetime
+import logging
+from lxml import etree
+
+from odoo.tests import tagged
+from odoo.addons.l10n_it_edi_sdicoop.tests.test_edi_xml import TestItEdi
+
+_logger = logging.getLogger(__name__)
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestItEdiReverseCharge(TestItEdi):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        def get_tags(*tag_codes):
+            return cls.env['account.account.tag'].search([('applicability', '=', 'taxes'), ('name', 'in', tag_codes)]).ids
+
+        def rep_lines(*lines):
```

Comment:
What do "rep_lines" means? Repartition lines?
Maybe add a comment or rename the function?